guides/common/modules/proc_configuring-pulpcore-api-workers.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Configuring_Pulpcore_API-Workers_{context}"]
+= Configuring Pulpcore API Workers
+
+The pulpcore-api workers respond to incoming API requests.
+Fewer API workers consume less memory and results in better performance.
```

Comment:
This is questionable advice. Not enough workers means you can't serve the requests in parallel. Taking your advice to the extreme means I simply have 1 worker. But with enough clients, you can saturate that and performance is lower than it could be.

The API service is used by Foreman (via Katello), but also by container content. If your tests didn't test any container workloads then you're only testing a fraction of what it needs to serve.

Reply:
@Imaanpreet Is this something you've observed? Do you have any numbers on this?